cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -245,6 +250,23 @@ func generateMountsCommonInstallRecover(recoverySystem string) (allMounted bool,
 		return false, nil
 	}
 
+	// 1a. measure model
+	err = stampedAction(fmt.Sprintf("%s-model-measure", recoverySystem), func() error {
+		return measureWhenPossible(func(tpm *secboot.TPMConnection) error {
+			// TODO:UC20: consider doing the measurement later and
+			// getting the model from opened system seed
```

Comment:
I'm not sure I understand this comment, this is reading from "ubuntu-seed", what would be the difference?

Reply:
A chance for potential refactor. Couple lines down we have `recoverySystemEssentialSnaps()` which opens the system seed. The seed itself has some convenience methods for accessing the model.

Reply:
the model at this point is not verified